.github/fabricbot.json

```diff
@@ -3687,7 +3687,7 @@
             {
               "name": "bodyContains",
               "parameters": {
-                "bodyPattern": "(aot)/i",
+                "bodyPattern": "([Aa][Oo][Tt])+",
```

Comment:
I think `[Aa][Oo][Tt]` would work, but you could add word boundaries if you were feeling really fussy.  I don't think the parens or `+` are needed.